pkg/controller/certificates/requestmanager/requestmanager_controller_test.go

```diff
@@ -467,7 +467,7 @@ func TestProcessItem(t *testing.T) {
 			),
 			requests: []runtime.Object{
 				gen.CertificateRequestFrom(bundle1.certificateRequest,
-					gen.SetCertificateRequestName("test-1"),
+					gen.SetCertificateRequestName("test"),
```

Comment:
I have absolutely no idea why this name has to change to make the test pass. 🤔

Reply:
See https://github.com/cert-manager/cert-manager/pull/7869#discussion_r2245192558.